src/server/shell/decoration/renderer.cpp

```diff
@@ -49,6 +56,33 @@ uint32_t const default_unfocused_background = color(0x80, 0x80, 0x80);
 uint32_t const default_focused_text         = color(0xFF, 0xFF, 0xFF);
 uint32_t const default_unfocused_text       = color(0xA0, 0xA0, 0xA0);
 
+struct FontPath
+{
+    char const* filename;
+    std::vector<char const*> prefixes;
+};
+
+FontPath const font_paths[]{
```

Comment:
Could we make this a CMake variable to let distros handle this as per their own policies/locations?

Reply:
Would that be able to work by default? I don't know of a standard way to query the build system for the distro policy.

Or would anyone building from source have to deal with specifying the font location as an extra step? (Doing the same sort of polling in CMake is a bad idea as the build system need not have fonts installed to be detected.)

There's also the consideration of snaps running on a variety of host distros. "Classic" snaps *could*, if we look around for them, pick up fonts from the host system.

Reply:
Maybe we should use [`FcFontMatch()`](https://www.freedesktop.org/software/fontconfig/fontconfig-devel/fcfontmatch.html)?